plugins/pagerduty/api-report.md

```diff
@@ -87,4 +210,15 @@ export function TriggerButton(props: TriggerButtonProps): JSX.Element;
 //
 // @public (undocumented)
 export class UnauthorizedError extends Error {}
+
+// Warning: (ae-missing-release-tag) "User" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export type User = {
```

Comment:
These type names end up being extremely generic, which I think can hurt projects using this package down the line. Imagine 50 plugins similar to this installed in a project where you want to autocomplete an import, kinda sucks to have to figure out which of the 30 different `User` types that you actually are supposed to import.

What do you think by prefixing pretty much all these added types by `PagerDuty*`? `PagerDutyUser`, `PagerDutyServiceResponse`, etc.